### PR TITLE
first, partial attempt to merge mg26x to master

### PR DESCRIPTION
**Pull request by perrozzi**

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/examples/wplustest_5f_NLO/wplustest_5f_NLO_run_card.dat [hunk 1]
```diff
@@ -93,22 +92,26 @@
...
 #***********************************************************************
- 1.0, 2.0, 0.5 = rw_rscale ! muR factors to be included by reweighting
- 1.0, 2.0, 0.5 = rw_fscale ! muF factors to be included by reweighting
+ 1.0 = rw_rscale ! muR factors to be included by reweighting
+ 1.0 = rw_fscale ! muF factors to be included by reweighting
  True = reweight_scale ! Reweight to get scale variation using the 
             ! rw_rscale and rw_fscale factors. Should be a list of 
             ! booleans of equal length to dynamical_scale_choice to
             ! specify for which choice to include scale dependence.
+<<<<<<< HEAD
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/examples/wplustest_5f_NLO/wplustest_5f_NLO_run_card.dat]: conflict to be resolved

File: bin/MadGraph5_aMCatNLO/cards/examples/wplustest_4f_NLO/wplustest_4f_NLO_run_card.dat [hunk 2]
```diff
@@ -10,6 +10,9 @@
 #   Lines starting with a hash (#) are info or comments                *
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/examples/wplustest_4f_NLO/wplustest_4f_NLO_run_card.dat]: file to be crosschecked

File: bin/MadGraph5_aMCatNLO/gridpack_generation.sh [hunk 3]
```diff
@@ -76,6 +661,7 @@ fi
 if [ -z ${jobstep} ]; then
   jobstep=ALL
 fi
+<<<<<<< HEAD
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/gridpack_generation.sh]: conflict to be solved

File: bin/MadGraph5_aMCatNLO/runcmsgrid_LO.sh [hunk 4]
```diff
@@ -68,6 +59,8 @@ fi
 #########################################
 # FORCE IT TO PRODUCE EXACTLY THE REQUIRED NUMBER OF EVENTS
 #########################################
+<<<<<<< HEAD
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/runcmsgrid_LO.sh]: conflict to be solved

File: bin/MadGraph5_aMCatNLO/runcmsgrid_NLO.sh [hunk 5]
```diff
@@ -16,6 +16,7 @@ if [ -n "$4" ]
   then
   use_gridpack_env=$4
 fi
+<<<<<<< HEAD
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/runcmsgrid_NLO.sh]: conflict to be solved

File: bin/MadGraph5_aMCatNLO/submit_cmsconnect_gridpack_generation.sh [hunk 6]
```diff
@@ -1,7 +1,12 @@
 #!/bin/bash
 
+<<<<<<< HEAD
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/submit_cmsconnect_gridpack_generation.sh]: conflict to be solved

File: bin/MadGraph5_aMCatNLO/submit_condor_gridpack_generation.sh [hunk 7]
```diff
@@ -1,10 +1,18 @@
 #!/bin/bash
+<<<<<<< HEAD
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/submit_condor_gridpack_generation.sh]: conflict to be solved

**Comment by perrozzi** [comment]
@AndreasAlbert can you please check?

**Comment by perrozzi** [comment]
ping

**Comment by AndreasAlbert** [comment]
@perrozzi PR here #1767

**Pull request closed without merging**